src/NuGet.Indexing/IndexingConfiguration.cs

```diff
@@ -9,7 +9,7 @@ namespace NuGet.Indexing
     /// <summary>
     /// Provides configuration for Indexing.
     /// </summary>
-    public class IndexingConfiguration : Configuration
+    public class IndexingConfiguration : NuGet.Services.Configuration.Configuration
```

Comment:
>uGet.Services.Configuration. [](start = 42, length = 28)

Is this change intended?

Reply:
Without the change it fails to build with 
> error CS0118: 'Configuration' is a namespace but is used like a type
I'm not sure how it works in our main branches.